docs/configuration/openconfig.md

```diff
@@ -156,6 +156,110 @@ provider smash
 provider sysdb
 ```
 
+## Syslog messages with OpenConfig
+
+EOS configuration:
```

Comment:
Starting EOS 4.27.2F `accounting requests` can be enabled to account records for gNMI/gNOI RPCs:

Reply:
done